src/components/StarRating.ts

```diff
@@ -36,15 +38,29 @@ export class StarRating extends Component<StarRatingProps, {}> {
         this.fractions = readOnly ? 2 : 1;
         this.stop = this.props.maximumStars;
 
-        return createElement("div", {
-            className: classNames("widget-star-rating", this.props.className),
-            style: this.props.style
-        },
+        const customStyle: object | undefined = this.props.starSize === "custom"
+            ? { fontSize: `${this.props.starSizeCustom}px` }
+            : undefined;
+        const EmptyStar = createElement("div", {
```

Comment:
Should not start with a capital is this is not a class or type nor interface `emptyStar `

Reply:
Was done